include/reporters/catch_reporter_bases.hpp

```diff
@@ -18,13 +18,16 @@
 #include <cassert>
 #include <memory>
 #include <ostream>
+#include <sstream>
```

Comment:
This include shouldn't be here anymore.

Otherwise if you squash the commits into one, this is ready for merging.

Reply:
Great. I'll update later today